app/script/auth/component/ClientItem.tsx

```diff
@@ -112,16 +136,13 @@ class ClientItem extends React.Component {
 
   resetState = () => this.setState(ClientItem.initialState);
 
-  wrappedOnClick = event => {
+  wrappedOnClick = (event: React.MouseEvent<HTMLDivElement>) => {
     this.resetState();
     this.props.onClick(event);
   };
 
-  handleSubmit = event => {
+  handleSubmit = (event: React.FormEvent) => {
     event.preventDefault();
-    if (this.props.isFetching) {
-      return;
-    }
```

Comment:
Where do we handle this now?

Reply:
I just copied over the functionality that was already present: https://github.com/wireapp/wire-webapp/blob/2018-10-02-08-03/app/script/auth/component/ClientItem.jsx#L122-L124

Reply:
This was a false friend. We never handled this state so it's better to remove it.